src/main/primary_entry/html/index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="description" content="Zoomable Tesla supercharger map with adjustable range indicator.">
     <link href="/service/supercharge/feed/atom.xml" type="application/atom+xml" rel="alternate"
           title="Supercharger Status Changes ATOM Feed"/>
-
+    <a rel="me" href="https://mastodon.social/@superchargeinfo"></a>
```

Comment:
Does this actually work for Mastodon verification as an empty link in the <head>? Or should we add a link with text within the About page (~line 615)?

Reply:
This is recommended way to verify ownership of a link on Mastodon. I updated it on the live site and the green verification mark appeared on our Mastodon account:

<img width="579" alt="Screenshot 2023-09-11 at 7 44 47 AM" src="https://github.com/supercharge-info/supercharge.info/assets/130219/c0ffb8d8-4e15-4818-996f-7de543534400">

You are right though, we should add a link to the About page.